python/tests/test_flows_file.py

```
@@ -0,0 +1,570 @@
+
+        mock_client.flows.trigger_with_file.side_effect = Exception("API Error: Invalid URL format")
+
+        with pytest.raises(Exception, match="API Error: Invalid URL format"):
+            process_remote_image()
+
+    @patch.dict(os.environ, {
+        "WORQHAT_API_KEY": "test-api-key"
+    })
+    @patch('endpoints.flows_file.Worqhat')
+    @patch('builtins.open', new_callable=mock_open, read_data=b"fake file content")
+    def test_process_document_with_params_workflow_not_found_error(self, mock_file_open, mock_worqhat_class):
```

Ruff (0.14.1)

Unused method argument: `mock_file_open`

(ARG002)